### ISSUE TITLE
Slim (console only) Debian build for Beaglebone Black that works with LEDscape?

### ISSUE THREAD

**Issue by seriousmumbo** [Yona-Appletree/LEDscape#25]
https://groups.google.com/d/msg/beagleboard/GANFPr7BtVo/ubCbAW2cSysJ

Hi, thought I'd cross post here as I imagine some of you have the same issue.  Anyone here found a slim console only image with Debian (or Ubuntu) that plays well with LEDscape?

**Comment by RGB-123** [comment]
Ryan, it sounds like you may have worked this out.  However, I think you simply needed to disable the HDMI in the uEnv.txt file.

**Comment by seriousmumbo** [comment]
Yes, kernel bone70 works with HDMI disabled.  I'm curious as to why this is necessary?  I like the HDMI =).

**Comment by mykolasmith** [comment]
It's because LEDscape uses those GPIO pins that would otherwise be used for HDMI. You can't have your cake and eat it too ;)

**Comment by seriousmumbo** [comment]
Speaking of tradeoffs: I noticed that the ledscape dtb breaks sd card hotplug.

**Comment by seriousmumbo** [comment]
Is the .dts source for the dtb available?

**Comment by seriousmumbo** [comment]
I created a new dtb, it should work well with the newer kernels (tested with bone70).  Advantages include: sdcard hotpug working, hwrng, enhanced quadrature encoder pulse decoder, bandgap (chip temp) sensor, GPS cape and much much more:

https://github.com/seriousmumbo/ledscape-dtb

My use case (one 24 pixel strip) is probably more limited than yours, so testing is in order.
